Gds/src/fprime_gds/flask/default_settings.py

```diff
@@ -28,7 +28,7 @@
 # Gds config setup
 GDS_CONFIG = fprime_gds.common.utils.config_manager.ConfigManager()
 
-for dir in [LOG_DIR, UPLOADED_UPLINK_DEST, UPLOADS_DEFAULT_DEST, DOWNLINK_DIR]:
-    os.makedirs(dir, exist_ok=True)
+for directory in [LOG_DIR, UPLOADED_UPLINK_DEST, UPLOADS_DEFAULT_DEST, DOWNLINK_DIR]:
```

Comment:
Me??? Improper variable names???  #blame @LeStarch

Reply:
haha, without pylint stuff like this happens